src/main/java/edu/jhu/clueless/ClientAction.java

```diff
@@ -0,0 +1,44 @@
+package edu.jhu.clueless;
```

Comment:
No, there will be additional fields. The class is just not finished yet and
is only a start here. I have more added in what I'm currently working on.
The action string is just the high level actions of chat, accuse, move,
suggest, etc.
On Apr 20, 2016 9:35 PM, "Rebecca Butterfield" <notifications@github.com>
wrote:

> In src/main/java/edu/jhu/clueless/ClientAction.java
> <https://github.com/ch34/ClueLess/pull/2#discussion_r60515024>:
>
> > @@ -0,0 +1,44 @@
> > +package edu.jhu.clueless;
>
> how will this transfer things like the location a player wants to move to
> or a card in their suggestion? is all this meant to be captured in the
> action string?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ch34/ClueLess/pull/2/files/bc365c260882528e2677608530348f7b7bb7180d#r60515024>
>